### PR TITLE
UCP/CONTEXT: Show warning about non-existing transport

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5905/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5906/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5908/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask @brminich pls take a look

File: src/ucp/core/ucp_context.c [hunk 1]
```diff
@@ -280,15 +280,19 @@ static unsigned ucp_tl_alias_count(ucp_tl_alias_t *alias)
...
+    if ((cfg_index = ucp_str_array_search(tls, count, tl_name, NULL)) >= 0) {
+        *tl_cfg_mask |= UCS_BIT(cfg_index);
         ucs_trace("enabling tl '%s'%s", tl_name, info);
         return 1;
-    } else if (ucp_str_array_search(tls, count, tl_name, "aux") >= 0) {
+    } else if ((cfg_index = ucp_str_array_search(tls, count, tl_name, "aux")) >= 0) {
         /* Search for tl names with 'aux' suffix, such tls can be
          * used for auxiliary wireup purposes only */
-        *flags |= UCP_TL_RSC_FLAG_AUX;
+        *rsc_flags |= UCP_TL_RSC_FLAG_AUX;
```
Comment (brminich) [inline src/ucp/core/ucp_context.c]: minor: `|=` alignment

**Pull request merged**